core/settings.py

```
@@ -1126,106 +1151,13 @@ class Offline(Base):
-            #     "rest_framework.throttling.AnonRateThrottle",
-            # ],
-            "DEFAULT_THROTTLE_RATES": {"anon": "1/minute", "user": "6/minute"},
-            "DEFAULT_PARSER_CLASSES": [
-                "rest_framework.parsers.JSONParser",
-            ],
-            "DEFAULT_FILTER_BACKENDS": [
-                "django_filters.rest_framework.DjangoFilterBackend",
-            ],
-            "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
-        },
+        _INSTALLED_APPS_CORE + ["debug_toolbar"] + _INSTALLED_APPS_PROJECT,
```

Ruff (0.15.14)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)